packages/runtime/test/services/LocalGateway.spec.ts

```diff
@@ -55,21 +56,24 @@ describe('services/LocalGateway', () =>
     {
         it('should find and run a procedure from a node', async () =>
         {
-            const firstResult = await gateway.run('second', Version.DEFAULT, new Map(), new Map());
+            const request = new Request('second', Version.DEFAULT, new Map(), new Map());
+            const response = await gateway.run(request);
 
-            expect(firstResult).toBe('first');
+            expect(response.result).toBe('first');
         });
 
         it('should find and run a procedure from a node that calls a procedure on another node', async () =>
         {
-            const result = await gateway.run('third', Version.DEFAULT, new Map(), new Map());
+            const request = new Request('third', Version.DEFAULT, new Map(), new Map());
+            const response = await gateway.run(request);
 
-            expect(result).toBe('fourth');
+            expect(response.result).toBe('fourth');
         });
 
         it('should not run a non-existing procedure', async () =>
         {
-            const run = async () => gateway.run('nonExisting', Version.DEFAULT, new Map(), new Map());
+            const request = new Request('nonExisting', Version.DEFAULT, new Map(), new Map());
+            const run = async () => gateway.run(request);
```

Comment:
I expect this name to be `response` to be consistent with the other tests

Reply:
You're right about the different approach, as it indeed contains an anonymous run function. I'll close the comments as Resolved